snuba/datasets/storages/errors_common.py

```diff
@@ -128,6 +128,7 @@
 
 prewhere_candidates = [
     "event_id",
+    "trace_id",
```

Comment:
What's the reason for including this change here?

Reply:
Trace id is useful in the prewhere just like it is on the transactions table as it is a very high selectivity column. IT is not strictly related with the event_id improvement. In the first version I was adding trace_id to the UUIDColumnProcessor as well, so I just went on and fixed trace_id everywhere. But then I removed it from UUICColumnProcessor and I jest left it here as it is still useful

Reply:
Yeah but we're getting that column out of contexts now since it's not promoted yet. So wouldn't this need to be `contexts[trace_id]` in order to have any effect?

Reply:
I'm actually not sure if this is used anywhere today. Even if it were, the column name registered in Sentry is `contexts[trace_id]` not `trace_id`, and we'd never remap it into `trace_id` so I don't think it'd ever be picked as a prewhere candidate even if it's on this list.

Reply:
ok, will remove for now